app/views/users/_organization_user.html.erb

```diff
@@ -30,8 +31,8 @@
         </ul>
       </div>
     <% end %>
-    <% if current_user.has_role?(Role::ORG_ADMIN, current_organization) && user.has_role?(Role::ORG_ADMIN, current_organization) %>
-      <%= edit_button_to demote_to_user_organization_path(user_id: user.id),
+    <% if current_user.is_admin?(current_organization) && user.has_role?(Role::ORG_ADMIN, current_organization) %>
```

Comment:
The first part of this condition can be removed. Essentially this is equivalent to `if (x || y) && x)` which can be replaced with `if x`.